src/frameworks/angular/__tests__/__snapshots__/angularURLReplacer.test.ts.shapshot.json

```diff
@@ -1,4 +1,6 @@
 {
   "0deb0dd1": "\n@Component({\n  selector: \"app-root\",\n  template : require(\"./hello.html\"),\n  styles : [require(\"./foo.css\"),require(\"bar.scss\")],\n})\n",
-  "0deb0dd0": "\n@Component({\n  selector: \"app-root\",\n  template : require(\"./hello.html\"),\n  styles : [require(\"bar.scss\")],\n})\n"
+  "0deb0dd0": "\n@Component({\n  selector: \"app-root\",\n  template : require(\"./hello.html\"),\n  styles : [require(\"bar.scss\")],\n})\n",
+  "04900d8c": "\n@Component({\n  selector: \"app-root\",\n  template : require(\"./hello.html\"),\n  styles : [require(\"./foo.css\"),require(\"bar.scss\")],\n})\n",
+  "04900d8b": "\n@Component({\n  selector: \"app-root\",\n  template : require(\"./hello.html\"),\n  styles : [require(\"bar.scss\")],\n})\n"
```

Comment:
Could you discard those changes? It looks like the wrong hash is generated

Reply:
Sure